okio/src/commonMain/kotlin/okio/-Util.kt

```diff
@@ -91,6 +91,8 @@ internal fun arrayRangeEquals(
 }
 
 internal fun Byte.toHexString(): String {
+  if (this == 0.toByte()) return "0x00"
```

Comment:
is this an optimization?

Reply:
No, I haven't benchmarked it. Happy to remove if you think it's not needed.

Reply:
I don’t like these. Here’s why!
https://smallwig.blogspot.com/2010/03/little-optimization-that-couldnt.html

Reply:
Good to know! Thanks for the learning!